src/Pages/Auth/Login.jsx

```diff
@@ -1,9 +1,84 @@
-import React from 'react'
+import React, { useState } from "react";
+import "../Auth/Auth.css";
+import { Link, useLocation, useNavigate } from "react-router-dom";
+import axios from "axios";
+import { ToastMsg } from "../../components";
+import { useAuth } from "../../context/Auth/context";
+import { LoginHandler } from "../../context/Auth/util";
 
 function Login() {
+  const { authDispatch } = useAuth();
+  const location = useLocation();
+  const navigate = useNavigate();
+
+  const [loginData, setLoginData] = useState({ email: "", password: "" });
+
+  const handleChange = (e) => {
+    const inputName = e.target.name;
+    const value = e.target.value;
+    setLoginData({ ...loginData, [inputName]: value });
+  };
+
+  const testCredentials = {
+    email: "vaibhavmahalle95@gmail.com",
+    password: "Vaibhav123",
+  };
+
+  const setLogin = () => {
+    setLoginData(testCredentials);
+  };
+
+  const handleLogin = async (e) => {
+    e.preventDefault();
+    // console.log("getting called")
```

Comment:
remove the commented code

Reply:
thank you for the suggestion will make the necessary changes.